cmd/root.go

```diff
@@ -67,32 +67,27 @@ func newRootCommand(gs *state.GlobalState) *rootCommand {
 
 func (c *rootCommand) persistentPreRunE(cmd *cobra.Command, args []string) error {
 	var err error
-
-	c.loggerStopped, err = c.setupLoggers()
+	c.loggerStopped, err = c.setupLoggers(c.globalState.Ctx)
```

Comment:
I think is relevant to mention that moving `setupLoggers` directly to the execute method could simplify the current synchronization process, but it would impact how it is integrated with Cobra. Something good for the next PR.